sonar-core/src/main/java/org/sonar/core/platform/ComponentContainer.java

```diff
@@ -298,13 +302,7 @@ public ComponentContainer removeChild(ComponentContainer childToBeRemoved) {
   }
 
   private ComponentContainer removeChildren() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "removeChildren" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)